rafay/resource_project.go

```diff
@@ -279,6 +279,10 @@ func expandProjectSpec(p []interface{}) (*systempb.ProjectSpec, error) {
 		obj.DriftWebhook = expandProjectDriftWebhook(v)
 	}
 
+	if v, ok := in["sync_excluded_namespaces"].([]interface{}); ok {
```

Comment:
can you update https://github.com/RafaySystems/terraform-provider-rafay/blob/master/docs/resources/project.md doc also..?

Reply:
okay, will make that change.